specs/interop/shared-lockbox-upgrade.md

```diff
@@ -0,0 +1,93 @@
+# Shared Lockbox - Upgrade and migration process
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Batch transaction process](#batch-transaction-process)
+- [Future Considerations / Additional Notes](#future-considerations--additional-notes)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+Based on the decision that a chain joining the dependency set is an irreversible process,
+the on-chain chains list is simplified by assuming that joining the Shared Lockbox is
+equivalent to joining the op-governed dependency set.
+
+The migration process consists of four main points:
+
+- `OptimismPortal` code upgrade (includes the `SharedLockbox` integration)
+- Move ETH liquidity from `OptimismPortal` to `SharedLockbox`
+- Set `SuperchainConfig` as dependency manager on `SystemConfig`
+- Add the chain to the op-governed dependency set
+
+The migration process needs some prerequisites:
+
+- `SharedLockbox` deployed
+- `SuperchainConfig` upgraded to manage the dependency set
+
+**`OptimismPortal` code upgrade**
```

Comment:
This one feels like the text could be divided into bulletpoints the way the text is written, perhaps it adds more clarity, could be good to check